taccsite_cms/templates/djangocms_blog/base.html

```diff
@@ -14,13 +14,21 @@
 {% endblock meta %}
 
 {% block content %}
+
```

Comment:
Eventually, will we be able top drop the `if` blocks for the version check?
If not, it might get complicated trying to track the v1/v2 files if the nomenclature does not directly reflect it (eg. 'site_cms/css/build/site.app.blog.css' vs 'site_cms/css/build/app.blog.css' does not intuitively indicate a versioning).

I may not be tracking this fully and this may be mitigated already through config management, etc.